pod/playlist/views.py

```diff
@@ -230,11 +233,13 @@ def playlist_edit(request, playlist):
 
 
 def playlist_add(request, playlist):
-    """Add a video (in POST) to the playlist. AJAX only."""
-    if request.is_ajax():
+    if is_ajax(request):
```

Comment:
il faudrait laisser les pydocs sur les fonctions ("""Add a video (in POST) to the playlist.""")